requirements.txt

```diff
@@ -0,0 +1,2 @@
+numpy
```

Comment:
might be worth investigating that it works for `numpy` version 2. Maybe freeze version if not.

Reply:
I just upgraded to latest NumPy and can confirm that everything still works.